angular/src/assets/img/.gitkeep

```diff
@@ -0,0 +1 @@
+@AngularClass
```

Comment:
Om de `img` folder te houden, op aanraden van Ruurd, maar misschien een beetje onnodig. Die in `src/assets/css` kan ook wel weg.
Aan de andere kant misschien ook we handig, misschien moet de folder zo heten omdat het build proces ervan uitgaat?